README.md

```diff
@@ -0,0 +1,138 @@
+# XTT-CPP: C++ wrappers for the XTT IoT security protocol
+
+This project provides two C++ APIs for the C-based XTT
+reference implementation [here](https://github.com/xaptum/xtt):
+- `libxtt-cpp`:
+  - A mostly-one-to-one C++ version of the C API provided by [here](https://github.com/xaptum/xtt)
+- `libxtt-asio`:
+  - A higher-level API that builds upon `libxtt-cpp` and uses the [Boost ASIO](https://www.boost.org/doc/libs/1_67_0/doc/html/boost_asio.html)
+    perform all necessary network I/O asynchronously
+
+# Project Status
+[![Build Status](https://travis-ci.org/xaptum/xtt-cpp.svg?branch=master)](https://travis-ci.org/xaptum/xtt-cpp)
+
+## Installation
+
+`xtt-cpp` is available for the following distributions. It may also be
+built from source.
+
+### Debian (Jessie or Stretch)
+
+``` bash
+# Install the Xaptum API repo GPG signing key.
+apt-get adv --keyserver keyserver.ubuntu.com --recv-keys c615bfaa7fe1b4ca
+
+# Add the repository to your APT sources, replacing <dist> with either jessie or stretch.
+echo "deb http://dl.bintray.com/xaptum/deb <dist> main" > /etc/apt/sources.list.d/xaptum.list
+
+# Install the library.
+sudo apt-get install libxtt-cpp-dev libxtt-asio-dev
+```
+
+### Homebrew (MacOS)
+
+``` bash
+# Tap the Xaptum Homebrew repository.
+brew tap xaptum/xaptum
+
+# Install the library.
+brew install xtt-cpp xtt-asio
+```
+
+## Installation from Source
+
+### Build Dependencies
+
+* CMake (version 3.0 or higher)
+* A C99-compliant compiler
+
+* [XTT](https://github.com/xaptum/xtt) (version 0.6.0 or higher)
```

Comment:
`xtt` is the only dependency, now right?

And `boost::asio` for `xtt-asio`.

Reply:
Yea, sorry, I deleted those others earlier, and somehow they snuck back in.